CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Next Release
+## On Demo
 ### Features
 - Refill prescriptions in edit medicine screen
 - [In Progress: 11 Nov 2020] Migrate app screens to use ViewBinding
```

Comment:
Should we extract out these in-progress items into the next release section?

Reply:
Ah yes, I'll do this.